server/routes/github/github_app.py

```
@@ -0,0 +1,1052 @@
+    """Connection status for the GitHub connector (hybrid-aware, with reconcile)."""
+    app_runtime_ready = bool(flask.current_app.config.get("GITHUB_APP_ENABLED"))
+    app_branch_active = is_app_enabled() and app_runtime_ready
+
+    if app_branch_active:
+        _reconcile_user_installations(user_id)
+
+    app_username: str | None = None
+    if app_branch_active:
+        try:
+            with db_pool.get_admin_connection() as conn:
+                with conn.cursor() as cur:
```

Ruff (0.15.12)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)